hedera-node/test-clients/scripts/diff-testing/versioned_output_file.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
```

Comment:
Just occurred to me, we probably need all of these files to have the license header

Reply:
It may be worth checking my assumption. We have other .py files in the repo that don't have this header, so maybe it's ok??